src/main/kotlin/no/nav/syfo/varsel/esyfovarsel/KafkaEsyfovarselConfig.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.syfo.varsel.esyfovarsel
+
+import no.nav.syfo.config.kafka.KafkaAivenConfig
+import no.nav.syfo.varsel.esyfovarsel.domain.EsyfovarselHendelse
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.kafka.annotation.EnableKafka
+import org.springframework.kafka.core.DefaultKafkaProducerFactory
+import org.springframework.kafka.core.KafkaTemplate
+import org.springframework.kafka.core.ProducerFactory
+
+@EnableKafka
+@Configuration
+class KafkaEsyfovarselConfig(
```

Comment:
Jeg synes egentlig `EsyfovarselKafkaConfig` leser bedre enn `KafkaEsyfovarselConfig`. Jeg tenker som så at det er en kafka-config av typen Esyfovarsel. Det kan være god praksis å navngi det sånn at man har den mest generiske typen til slutt, og at det blir mer og mer spesifikt mot venstre. 

F.eks. i JDK'en så har man `OutputStream` og `InputStream`, med subklasser som f.eks `FilterInputStream`, 
`BufferedInputStream` og `ByteArrayInputStream`.

Vi kan evt rename `KafkaDialogmotekandidatConfig` til `DialogmotekandidatKafkaConfig`

Reply:
REnamed denne lonfigen akkurat for å bli i  tråd med de andre konfigene som du har nevnt. 
Reverterer denne og renamer de andre.